.github/workflows/nodejs.yml

```diff
@@ -60,7 +63,7 @@ jobs:
         # forceExit: something is not cleaning up - likely firebase-app that needs to be deleted, investigate
         run: |
           npm install -g firebase-tools@latest
-          firebase emulators:exec --only database,firestore,storage --project sample 'npm run test:coverage -- --forceExit'
+          firebase emulators:exec --only database,firestore,storage --project sample 'npm run test:coverage -- --forceExit --logHeapUsage'
         env:
```

Comment:
Is this still needed now we know the memory usage?

Reply:
Hmm technically no - but it might help keep our tests in check, if we notice the size spiking? Granted no one will ever notice it so maybe not worth keeping. I can go either way.

Argument for keeping: future debugging is quicker
Argument for dropping: not really needed for the coverage test

wdyt?